src/Generator/Passes/GenerateSymbolsPass.cs

```diff
@@ -55,7 +55,7 @@ private void GenerateSymbols()
                             !m.IsImplicit && !m.IsDeleted && !m.IsDefaulted;
                         if (specialization.Methods.Any(m => m.IsInvalid && exportable(m)))
                             foreach (var method in specialization.Methods.Where(
-                                m => m.IsGenerated && exportable(m)))
+                                m => m.IsGenerated && m.InstantiatedFrom.IsGenerated && exportable(m)))
```

Comment:
As I warned you at the beginning, InstantiatedFrom may be null. You do not check it as I asked and the result is precisely https://ci.appveyor.com/project/tritao/cppsharp/build/1.0.3005/job/896ctkux248jd7y8#L497 . In addition, I understand you prefer to work on Linux but our tests can run there too. I have a feeling you don't ensure they do before sending code, not just in this particular pull request. Please correct me if I'm wrong. If not, please understand that valid tests mean work less work not just for me but for you as well.

Reply:
NUnit actually crashes on my machine. Sorry about that. Please see if new commit is any better.